tests/Makefile

```diff
@@ -31,7 +31,8 @@ BCL_TEST_SUITES=mscorlib \
 	Mono.Data.Tds \
 	System.IO.Compression \
 	System.IO.Compression.FileSystem \
-	Mono.CSharp
+	Mono.CSharp \
+	System.Security
```

Comment:
If you add a backslash for the last entry, any subsequent diffs will only have a single green line.

You'd also have to add an empty line after the list:

```
    Mono.CSharp \
    System.Security \

ALL_TEST_SUITES=...
```